plugins/nanospell/plugin.js

```diff
@@ -429,23 +416,36 @@
 				triggerSpelling((spellFastAfterSpacebar && (ch8r === CHARCODES.SPACE || ch8r === CHARCODES.LF || ch8r === CHARCODES.CR)), target)
 			}
 
+			function findNearestParentBlock(element) {
+				var elementPath = new CKEDITOR.dom.elementPath(element),
+					elements = elementPath.elements,
+					i;
+
+				for (i in elements) {
```

Comment:
Worried that this is actually doing "findRandomBlockInPath" in some browser, somewhere.